stack.yaml

```diff
@@ -1,17 +1,17 @@
 # stack yaml for ghc builds
 
-resolver: lts-14.27
+resolver: lts-15.3
 
 extra-deps:
   # --- Missing from Stackage --- #
   - ed25519-donna-0.1.1
   - hspec-golden-0.1.0.1
+  - direct-sqlite-2.3.26
+  - sbv-8.6
 
   # --- Forced Downgrades --- #
-  - sbv-8.2
-
-  # --- Forced Upgrades --- #
-  - trifecta-2.1
+  - megaparsec-7.0.5
```

Comment:
Down-pinned to avoid work up-front. We had started the MP8 port [in this PR](https://github.com/kadena-io/pact/pull/769).